Data/ByteString/Lazy.hs

```diff
@@ -807,9 +807,9 @@ splitWith p (Chunk c0 cs0) = comb [] (S.splitWith p c0) cs0
 -- | /O(n)/ Break a 'ByteString' into pieces separated by the byte
 -- argument, consuming the delimiter. I.e.
 --
--- > split '\n' "a\nb\nd\ne" == ["a","b","d","e"]
--- > split 'a'  "aXaXaXa"    == ["","X","X","X",""]
--- > split 'x'  "x"          == ["",""]
+-- > split 10 "a\nb\nd\ne" == ["a","b","d","e"]   -- fromEnum '\n' == 10
+-- > split 97  "aXaXaXa"   == ["","X","X","X",""] -- fromEnum 'a' == 97
+-- > split 120  "x"        == ["",""]             -- fromEnum 'x' == 120
```

Comment:
Could you please align strings by `"` and second `==` as well?

Reply:
sure